scripts/run_model_in_rocm_hvd_docker.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+MODEL="BERT-Large"
+#IMAGE="rocm/tensorflow:latest"
+IMAGE="rocm/tensorflow:rocm3.1-tf1.15-dev"
```

Comment:
does this image have horovad?

Reply:
I read your comment in the readme.

Reply:
Unfortunately, there is no public available container at this time. So this script is provided as a future reference here.